src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IDynamicInvocationExpression.cs

```diff
@@ -491,5 +491,490 @@ Type Arguments(0)
 
             VerifyOperationTreeAndDiagnosticsForTest<InvocationExpressionSyntax>(source, expectedOperationTree, expectedDiagnostics);
         }
+
+        [CompilerTrait(CompilerFeature.IOperation, CompilerFeature.Dataflow)]
+        [Fact]
+        public void DynamicInvocation_ControlFlowInReceiver()
+        {
+            string source = @"
+class C
+{
+    void M(C c1, C c2, dynamic d)
+    /*<bind>*/{
+        (c1 ?? c2).M2(d);
```

Comment:
>(c1 ?? c2).M2(d); [](start = 8, length = 17)

Please make sure we have a test without any control flow.

Reply:
Added `DynamicInvocation_NoControlFlow`

---
In reply to: [186835303](https://github.com/dotnet/roslyn/pull/26690#discussion_r186835303) [](ancestors = 186835303)